bcda-site-static/user_guide.md

```diff
@@ -624,3 +624,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
 	}
 }
 ```
+
```

Comment:
The user guide changes looks like a duplication of the work that was done in BCDA-1077 ( #225).  Is there anything different in here then in that work?

Reply:
I think the documentation in api.go is new.  That will add this to swagger

Reply:
yep just saw that. Other than the changes done to bcda/api.go I think everything else is a duplication. Im a do a pull now that #225 is merged and it should resolve.